### DISCUSSION TITLE
Static file serving from node_modules

### DISCUSSION THREAD

**Discussion by jonlambert** [vercel/next.js#39246] [Help]
Hi there,

I have a series of packages containing components and branding assets (fonts, logos, etc) that are imported into a number of different Next.js apps. 

How can I reliably serve up these static assets from a package in the `node_modules` directory without having to copy them into each site's `public/` directory? 

I found a similar [discussion from a couple of years ago](https://github.com/vercel/next.js/discussions/12844#discussioncomment-14380) that seems to recommend adding a webpack copy plugin call, but that presumably wouldn't support SWC and seems... dirty.

[Laravel's approach](https://laravel.com/docs/9.x/packages#public-assets) illustrates the kind of flow I'm imagining.

Any suggestions would be greatly appreciated!

**Comment by justintemps** [comment]
Hey @jonlambert , did you ever figure this out?
